### PR TITLE
Implement myuw-profile web component to the app

### PR DESCRIPTION
**Pull request by nogalpaulina**
### Breaking Change in 14.0.2

+ Replace existing profile menu with `myuw-profile` v1.6.1 from myuw-web-components library
+ Two files related to old profile menu have been removed: `components/portal/main/partials/username.html`
  and `buckyless/directives/username-menu.less`
+ Remove unused username elements
+ Upgrade myuw-help Web Component to version 1.5.3
+ Add tooltip to my-uw-help
+ Add role="toolbar" to myuw-app-bar

Review for security considerations

- [x] The change has been examined for security impact.

PR considerations checklist:

- [x] Updates `CHANGELOG.md` to reflect this PR's change.
- [x] This Contribution is under the terms of Individual [Contributor License Agreements][] (and also Corporate Contributor License Agreements to the extent applicable) appearing in the [Apereo CLA roster][].

[Apereo CLA roster]: http://licensing.apereo.org/completed-clas
[Contributor License Agreements]: https://www.apereo.org/licensing/agreements

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,18 @@ and this project adheres to
 
 + No changes yet
 
+## 14.0.2 - 2020-04-24
+
+### Breaking Change in 14.0.2
```
Comment (apetro) [inline CHANGELOG.md:15]: If it's a breaking change, then the major part of the major.minor.patch version number [should](https://semver.org/#spec-item-8) increment, so, 15.0.0.

File: components/css/buckyless/md-generated.less [hunk 2]
```diff
@@ -260,3 +230,8 @@ md-tooltip {
     }
   }
 }
+
+// Adjust z-index of tooltips
+.md-panel-outer-wrapper.md-panel-is-showing {
+  z-index: 54 !important;
+}
```
Comment (apetro) [inline components/css/buckyless/md-generated.less:237]: [The linter flags this missing end-of-file newline](https://travis-ci.org/github/uPortal-Project/uportal-app-framework/builds/679020979#L759). [editorconfig](https://editorconfig.org/) should be [automating this bit of style](https://github.com/uPortal-Project/uportal-app-framework/blob/master/.editorconfig#L28). Opportunity to configure local tools to honor editorconfig?

File: components/css/buckyless/header.less [hunk 3]
```diff
@@ -112,4 +112,23 @@ portal-header {
   .header__bell {
     display: inline-block;
   }
+
+  .header__guest-login-btn {
+    font-weight: 600;
+    &:hover, &:focus {
```
Comment (apetro) [inline components/css/buckyless/header.less:118]: [linter flags style issues with this less](https://travis-ci.org/github/uPortal-Project/uportal-app-framework/builds/679020979#L753).

File: components/portal/main/partials/main-menu.html [hunk 4]
```diff
@@ -50,7 +50,31 @@
...
+         aria-label="Log in to {{ portal.theme.title }}"
+         target="_self"
+         class="md-default header__guest-login-btn">
+        Log In
+      </md-button>
+
+    <!-- NOT GUEST MODE -->
+    <myuw-profile
+      ng-hide="sessionCtrl.guestMode"
+      aria-label="Log in to {{ portal.theme.title }}"
```
Comment (apetro) [inline components/portal/main/partials/main-menu.html:67]: This `aria-label` doesn't look right. This is the not-guest-mode, right, where the user is already logged in?

Comment (nogalpaulina) [reply components/portal/main/partials/main-menu.html]: Yes! Thank you, I changed it!

File: components/css/buckyless/md-generated.less (hunk 2 above)
Comment (nogalpaulina) [reply components/css/buckyless/md-generated.less:237]: I made the change in .editconfig!

File: components/css/buckyless/header.less [hunk 5]
```diff
@@ -112,4 +112,28 @@ portal-header {
...
+    font-weight: 600;
+
+    &:hover,
+    &:focus {
+      color: @white;
+      text-decoration: underline;
+      background-color: rgba(255, 255, 255, 0.11) !important;
+    }
+
+  }
```
Comment (apetro) [inline components/css/buckyless/header.less:126]: [linter flags this](https://travis-ci.org/github/uPortal-Project/uportal-app-framework/builds/679112260#L754): Unexpected empty line before closing brace.

File: components/portal/main/partials/header.html [hunk 6]
```diff
@@ -62,6 +63,32 @@
     </div>
   </myuw-help>
 
-  <username slot="myuw-profile" layout="row" layout-align="start center" hide-xs></username>
+  <!-- GUEST MODE ONLY -->
+  <md-button
```
Comment (cksteven) [inline components/portal/main/partials/header.html:67]: Adding one line here would fix (at least locally) the issue of guestMode!
```
ng-controller="SessionCheckController as sessionCtrl"
```

**Pull request merged**